src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
@NOYB The fact that the bundle is maintained by pkg is significant because pkg could verify that if we wanted, it doesn't need to be handled via the same mechanism. When restoring config.xml from another system, the bundle will already be there from pkg so we do not need to include it in config.xml. Let pkg worry about the bundle, let config.xml and pfSense worry about the contents of the other files. They each get verified by their source.

We'll have to agree to disagree if you can't see the logic behind that.